styles/modules/modals/_editListing.scss

```diff
@@ -240,13 +239,33 @@
         .inventoryTableWrap {
           overflow: auto;
           max-height: 500px;
+          position: relative;
         }
 
         table {
           border-spacing: 0;
           border-collapse: collapse;
           width: 100%;
 
+          thead tr td {
+            position: -webkit-sticky;
+            position: sticky;
+            top: 0px;
+            padding: 0px;
+            margin:0px;
+            border: none;
+            z-index:5;
+          }
+
+          thead tr td .stickyDiv {
+            width: inherit;
+            height: inherit;
+            background-color: white;
```

Comment:
When it comes to colors, we have planned the ability for the user to be able to switch themes. It hasn't been implemented yet, but the general idea is to not hard code colors in the css (outside of the themes file).

Basically, instead of hard-coded the colors here, you want to add certain classes to the respective elements.

So, instead of declaring the following in the css:

```
background-color: white;
border: 1px solid grey;
```

In the markup, you would want to add the `border clrBr clrP` classes to the element your targeting.